share/man/man5/src.conf.5

```diff
@@ -1,5 +1,5 @@
 .\" DO NOT EDIT-- this file is @generated by tools/build/options/makeman.
-.Dd November 13, 2023
+.Dd February 3, 2024
```

Comment:
I'd omit this generated file from the review. Other people's changes will just get in the way.